README.md

```diff
@@ -170,3 +172,69 @@ someinternalhost_IP = 10.128.0.8
 8. `shutdown your web service sudo systemctl stop pritunl`
 9. `sudo certbot certonly --standalone -d BastionIPAddress.nip.io`
 10. install created certificate to web application
+
+## Lecture Otus 06 Deploy test application
+
+* Install and setup gcloud with our account
+* Create host with gcloud help
+* Install ruby on this host
+* Instal MongoDB and launch
+* Deploy test application, launch and testing
+* testapp_IP = 34.168.221.121
+* testapp_port = 9292
+
+### Install gcloud
+
+1. `curl -O https://dl.google.com/dl/cloudsdk/channels/rapid/downloads/google-cloud-cli-400.0.0-linux-x86_64.tar.gz`
+2. extract archive.
+3. `sudo bash cd ./google-cloud/install.sh`
+4. `gcloud init` (use your project in account GCP)
+5. check installation
+    * `gcloud auth list`
+
+    * ```bash
+        gcloud compute instances create reddit-app\
+        boot-disk-size=10GB \
+        image-family ubuntu-1804-lts \
+        image-project=ubuntu-os-cloud \
+        machine-type=g1-small \
+        tags puma-server \
+        restart-on-failure
+        ```
+
+6. Or use actual instruction <https://cloud.google.com/sdk/docs/install-sdk>
+
+### Copy scripts to remote VM instance
+
+in your repo `bash copy_to_vm`
+
+### Install Ruby and Bundler
+
+* `ssh username@hostname` login to target host
+  
+* `bash /script/install_ruby.sh`
```

Comment:
(Here and below) Your scripts will not be copied to the `/script` directory rather they will be copied to `/home/appuser/` directory